etc/bento.env

```diff
@@ -58,12 +53,20 @@ BENTOV2_AUTH_LOGIN_REDIRECT_PATH=/*
 BENTOV2_AUTH_MEM_LIM=1G
 BENTOV2_AUTH_CPUS=2
 
+# Auth DB
+BENTO_AUTH_DB_CONTAINER_NAME=${BENTOV2_PREFIX}-auth-db
+BENTO_AUTH_DB="auth"
+BENTO_AUTH_DB_USER="auth_user"
+# BENTO_AUTH_DB_PASSWORD is set in local.env
```

Comment:
Should be in ./etc/bento_dev.env , if not set the auth service will not start and we have to look at the container logs to diagnose the problem.